ors-engine/src/main/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopperConfig.java

```diff
@@ -39,7 +40,18 @@ public static ORSGraphHopperConfig createGHSettings(ProfileProperties profile, E
         }
 
         ElevationProperties elevationProps = engineConfig.getElevation();
-        if (elevationProps.getProvider() != null && elevationProps.getCachePath() != null) {
+
+        if (!profile.getBuild().getElevation()) {
+            LOGGER.warn("Elevation is set to false.");
+        }
+        if (elevationProps.getProvider() == null) {
+            LOGGER.warn("Elevation provider is null.");
+        }
+        if (elevationProps.getCachePath() == null) {
+            LOGGER.warn("Elevation cache path is null.");
+        }
+
+        if (profile.getBuild().getElevation() && elevationProps.getProvider() != null && elevationProps.getCachePath() != null) {
```

Comment:
Add a variable e.g. boolean addElevation = true;
If any of the follwing if statements hits, switch it to false and print a warning.
The if clause in line 54 would then just be a single if(addElevation) ...